Lib/glyphsLib/interpolation.py

```diff
@@ -203,27 +208,12 @@ def add_masters_to_writer(ufos, axes, writer):
         for axis in axes:
             location[axis] = font.lib.get(
                 GLYPHS_PREFIX + axis + 'Value', DEFAULT_LOCS[axis])
-        master_data.append((font.path, family, style, location))
-
-    # pick a master to copy info, features, and groups from, trying to find the
-    # master with a base style shared between all masters (or just Regular) and
-    # defaulting to the first master if nothing is found
```

Comment:
Done in https://github.com/googlei18n/glyphsLib/commit/21293067128ae87be7b7664a9302a46b13a368de.